### PR TITLE
KEYCLOAK-5968 Move configuration of Infinispan server from Dockerfile to maven

### PR DESCRIPTION
**Pull request by tkyjovsk**

### CODE REVIEW

File: testsuite/performance/infinispan/pom.xml [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>performance-keycloak-infinispan-server</artifactId>
+    <name>Keycloak Performance TestSuite - Infinispan Server</name>
+    <packaging>pom</packaging>
+    
+    <properties>
+        <infinispan.groupId>org.infinispan.server</infinispan.groupId>
+        <infinispan.artifactId>infinispan-server-build</infinispan.artifactId>
+        <infinispan.version>8.2.8.Final</infinispan.version>
```
Comment (hmlnarik) [inline testsuite/performance/infinispan/pom.xml]: Can this use ${infinispan.version} from https://github.com/keycloak/keycloak/blob/master/pom.xml#L68?

File: testsuite/performance/infinispan/pom.xml [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+                        </goals>
+                        <configuration>
+                            <overWriteIfNewer>true</overWriteIfNewer>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>${infinispan.groupId}</groupId>
+                                    <artifactId>${infinispan.artifactId}</artifactId>
+                                    <version>${infinispan.version}</version>
+                                    <type>zip</type>
+                                    <outputDirectory>${project.build.directory}/dc1</outputDirectory>
```
Comment (hmlnarik) [inline testsuite/performance/infinispan/pom.xml]: Currently this extracts 2 infinispan servers into two different locations (dc1, dc2) and then updates clustered.xml just in the same way. Can you update it to extract it just once instead and then work on configuration clustered-dc*X*.xml instead, and run the appropriate configuration file in docker?

**Review by hmlnarik — CHANGES_REQUESTED** [review]
Overall lgtm, the way infinispan servers are prepared could be improved though.

**Comment by tkyjovsk** [comment]
I made changes based on previous comments. Infinispan servers in cross-DC scenario start up fine. I haven't tested the whole cross-dc deployment because of this blocker: KEYCLOAK-6411.

**Comment by tkyjovsk** [comment]
I tested this using workaround for KEYCLOAK-6411. (mariadb 10.3.3 instead of 10.3.4)

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**